package-lock.json

```diff
@@ -1,8 +1,16958 @@
 {
   "name": "razeedash-api",
   "version": "0.6.3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
The lockfileVersion changing from 1 to 2 suggests you updated locally while using Node 16 rather than Node 14, which is still specified in the `.travis.yml` file.  Moving from 14 to 16 might be a good idea, but should be separate from this PR (might have unintended consequences).